mqt/qcec/bindings.cpp

```diff
@@ -283,6 +291,8 @@ PYBIND11_MODULE(pyqcec, m) {
          "alternating_scheme"_a = "proportional", "profile"_a = "",
          "trace_threshold"_a = 1e-8, "fidelity_threshold"_a = 1e-8,
          "max_sims"_a = std::max(16U, std::thread::hardware_concurrency() - 2U),
+         "parameterized_tolerance"_a   = 1e-12,
+         "additional_instantiations"_a = 0U,
```

Comment:
Just as a reminder that this also might need to be touched. Same question as above.

Reply:
Done